### PR TITLE
Add optional customizable cancelBarButtonItem and doneBarButtonItem

### PR DESCRIPTION
**Pull request by jianyaoang**
Add optional UIBarButtonItems in ImagePickerConfigurable to customise appearance of cancelBarButtonItem and doneBarButtonItem.

### CODE REVIEW

File: Example/Pickle/CarousellImagePickerController.swift [hunk 1]
```diff
@@ -70,6 +70,10 @@ internal class CarousellImagePickerController: ImagePickerController {
 
 private struct CarousellTheme: ImagePickerConfigurable {
 
+    let cancelBarButtonItem: UIBarButtonItem? = UIBarButtonItem(barButtonSystemItem: .stop, target: nil, action: nil)
+
+    let doneBarButtonItem: UIBarButtonItem? = UIBarButtonItem(title: "Next", style: .plain, target: nil, action: nil)
```
Comment (bcylin) [inline Example/Pickle/CarousellImagePickerController.swift]: Would it be better to use `NSLocalizedString` like `imagePicker.button.camera`? so we can later update the translation.

File: Pickle/Classes/ImagePickerConfigurable.swift [hunk 2]
```diff
@@ -13,6 +13,14 @@ import UIKit
 /// The ImagePickerConfigurable protocol defines the customizable properties of ImagePickerController.
 public protocol ImagePickerConfigurable {
 
+    // MARK: - UINavigationItem
+
+    /// A custom bar button item displayed on the left (or leading) edge of the navigation bar when the receiver is the top navigation item.
+    var cancelBarButtonItem: UIBarButtonItem? { get }
+
+    /// A custom bar button item displayed on the right (or trailing) edge of the navigation bar when the receiver is the top navigation item.
+    var doneBarButtonItem: UIBarButtonItem? { get }
```
Comment (bcylin) [inline Pickle/Classes/ImagePickerConfigurable.swift]: Documented++

File: Example/Pickle/CarousellImagePickerController.swift (hunk 1 above)
Comment (bcylin) [inline Example/Pickle/CarousellImagePickerController.swift]: Just noticed that it's for the example app. It should be alright. 😅

Comment (jianyaoang) [inline Example/Pickle/CarousellImagePickerController.swift]: 👍

**Pull request merged**